src/pyodide/python-entrypoint-helper.ts

```diff
@@ -131,9 +131,14 @@ function makeHandler(pyHandlerName: string): Handler {
         'prep_python',
         async () => await preparePython()
       );
-
+      const handler = mainModule[pyHandlerName];
+      if (!handler) {
+        throw new Error(
+          `Python entrypoint "${MAIN_MODULE_NAME}" does not export a handler named "${pyHandlerName}"`
```

Comment:
We should check what the corresponding error for JS workers is and update this to match.